components/groups/modals/updateGroupModal.tsx

```diff
@@ -12,12 +12,51 @@ import { cosmos } from '@liftedinit/manifestjs';
 import { Any } from '@liftedinit/manifestjs/dist/codegen/google/protobuf/any';
 import env from '@/config/env';
 
-import { isValidManifestAddress, secondsToHumanReadable } from '@/utils/string';
+import { secondsToHumanReadable } from '@/utils/string';
 import { TrashIcon, PlusIcon } from '@/components/icons';
 import { MdContacts } from 'react-icons/md';
 import { TailwindModal } from '@/components/react/modal';
 import { Dialog } from '@headlessui/react';
 import { SignModal } from '@/components/react';
+import { duration, group as groupSchema } from '@/schemas';
+
+const updateFormSchema = groupSchema.metadataSchema
+  .shape({
+    threshold: Yup.number().min(1, 'Threshold must be at least 1').required().default(1),
+    votingPeriod: duration.schema
+      .required()
+      .test(
+        'min-total-time',
+        () => `Voting period must be at least ${secondsToHumanReadable(env.minimumVotingPeriod)}`,
+        value => duration.toSeconds(value) >= env.minimumVotingPeriod
+      )
+      .default(() => duration.fromSeconds(env.minimumVotingPeriod)),
+  })
+  .test(
+    'metadata-total-length',
+    'Total metadata length must not exceed 100000 characters',
+    function (values) {
+      const metadata = JSON.stringify({
+        title: values.title ?? '',
+        authors: values.authors ?? '',
+        details: values.details ?? '',
+      });
+      return metadata.length <= 100_000;
```

Comment:
Should we move this to `schemas/groups.ts` and re-use things like `MAXIMUM_GROUP_METADATA_JSON_LENGTH`?

Reply:
Just removed. There's no way this can be reached anyway as we limit every fields in the metadata. Maybe with 5 thousand authors...